cypress/integration/users/users.spec.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress" />
+import userData from '../../../public/data/eddiejaoude.json'
+
+const baseUrl = 'http://linkfree.eddiehub.org';
+
+context('Users', () => {
+    beforeEach(() => {
+      cy.visit(`${baseUrl}/eddiejaoude`,)
```

Comment:
Could this be done as a `click` from the homepage to check the navigating from the homepage to the user page works?